examples/js/exporters/GLTFExporter.js

```diff
@@ -48,6 +48,94 @@ var PATH_PROPERTIES = {
 	morphTargetInfluences: 'weights'
 };
 
+var DEFAULT_OPTIONS = {
+	mode: "glb",
+	trs: true,
+	onlyVisible: true,
+	truncateDrawRange: true,
+	animations: [],
+	forceIndices: false,
+	forcePowerOfTwoTextures: false
+};
```

Comment:
What's the purpose of exposing `DEFAULT_OPTIONS`, `getPaddedBufferSize()`, `getPaddedArrayBuffer()`, and `stringToArrayBuffer()` to global?

Reply:
I think we shouldn't expose unless there is any special reasons.